api/v01/entities/vrp_result.rb

```diff
@@ -44,6 +44,13 @@ class VrpResultSolutionRouteActivityDetails < Grape::Entity
       expose :timewindows, using: VrpResultSolutionRouteActivityDetailTimewindows, documentation: { is_array: true, desc: '' }
     end
 
+    class VRPResultSolutionRouteCosts < Grape::Entity
+      expose :fixed, documentation: { type: String, desc: 'Cost associated to the use of the vehicle' }
+      expose :time, documentation: { type: String, desc: 'Cost associated to the time dimension' }
+      expose :distance, documentation: { type: String, desc: 'Cost associated to the distance dimension' }
+      expose :value, documentation: { type: String, desc: 'Cost associated to the value dimension' }
+    end
```

Comment:
We don't want the total cost for ease of comparison (otherwise comparing two results becomes a bit of a chore) ?

Reply:
@braktar  it looks like, you forgot to add the `total`